pages/project-version/read/index.js

```diff
@@ -32,7 +32,9 @@ module.exports = settings => {
     res.locals.model = req.version;
     res.locals.static.project = req.project;
     res.locals.static.version = req.version.id;
-    res.locals.static.isGranted = req.project.status === 'active' && req.project.granted.id === req.version.id;
+    res.locals.static.isGranted = req.project.status === 'active' &&
+      req.project.granted.id === req.version.id &&
+      req.project.schemaVersion > 0;
```

Comment:
Y THO?

Can you just add a comment to explain this pls, because on reading the code it looks super weird.